test/e2e/common/setup-kubestellar.sh

```diff
@@ -109,14 +104,25 @@ esac
 : Install the core-chart
 :
 
-pushd "${SRC_DIR}/../../.."
-make kind-load-image
-make install-local-core-chart \
-  INSTALL_KUBEFLEX=false \
-  ITS_NAME=its1 \
-  DEFAULT_WDS_NAME=wds1 \
-  KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY=$KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY
-popd
+if [ "$use_release" = true ] ; then
+  kubestellar_latest_release=$(curl -sL https://github.com/kubestellar/kubestellar/releases/latest | grep "</h1>" | tail -n 1 | sed -e 's/<[^>]*>//g' | xargs)
+  helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+    --version ${kubestellar_latest_release//v} \
+    --set kubeflex-operator.install=false \
+    --set-json='ITSes=[{"name":"its1"}]' \
+    --set-json='WDSes=[{"name":"wds1"}]' \
+    --set verbosity.kubestellar=${KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY} \
+    --set verbosity.transport=${TRANSPORT_CONTROLLER_VERBOSITY}
+else
+  pushd "${SRC_DIR}/../../.."
+  make kind-load-image
+  make install-local-core-chart \
+    INSTALL_KUBEFLEX=false \
+    ITS_NAME=its1 \
+    DEFAULT_WDS_NAME=wds1 \
```

Comment:
Did not originate here, should be fixed in a separate PR: this variable is named wrong. It is simply the WDS name, it does not hold a default that might get overridden somewhere.

Reply:
Yes it's an awkward name. I can fix in a separate PR.
I'm also wondering if there is a need to use the `make install-local-core-chart` instead of a direct helm command pointing to the local `core-chart/` folder.
The second would be more clear, I think... Then do we need the `install-local-core-chart` target, with the repeated arguments?

Reply:
I implemented the change in https://github.com/kubestellar/kubestellar/pull/2473.
We will need to rebase.